### PR TITLE
chore: use new dbus of dock daemon

### PR DESCRIPTION
**Pull request by tsic404**
becase dock daemon has been moved into dock, update new dbus path

log: as title

### CODE REVIEW

File: src/dbusinterface/dbusdockinterface.cpp [hunk 1]
```diff
@@ -11,18 +11,226 @@
...
+}
+
+int DBusDockInterface::position()
+{
+    return qvariant_cast<int>(property("Position"));
+}
+
+void DBusDockInterface::setPosition(int value)
+{
+   setProperty("Position", QVariant::fromValue(value));
```
Comment (BLumia) [inline src/dbusinterface/dbusdockinterface.cpp:161]:
```suggestion
    setProperty("Position", QVariant::fromValue(value));
```

File: src/dbusinterface/dbusdockinterface.cpp [hunk 2]
```diff
@@ -11,18 +11,226 @@
...
+
+uint DBusDockInterface::showTimeout()
+{
+    return qvariant_cast<uint>(property("ShowTimeout"));
+}
+
+void DBusDockInterface::setShowTimeout(uint value)
+{
+
+   setProperty("ShowTimeout", QVariant::fromValue(value));
```
Comment (BLumia) [inline src/dbusinterface/dbusdockinterface.cpp:172]:
```suggestion
    setProperty("ShowTimeout", QVariant::fromValue(value));
```

File: src/dbusinterface/dbusdockinterface.cpp [hunk 3]
```diff
@@ -11,18 +11,226 @@
...
+}
+
+uint DBusDockInterface::windowSize()
+{
+    return qvariant_cast<uint>(property("WindowSize"));
+}
+
+void DBusDockInterface::setWindowSize(uint value)
+{
+   setProperty("WindowSize", QVariant::fromValue(value));
```
Comment (BLumia) [inline src/dbusinterface/dbusdockinterface.cpp:182]:
```suggestion
    setProperty("WindowSize", QVariant::fromValue(value));
```

File: src/dbusinterface/dbusdockinterface.cpp [hunk 4]
```diff
@@ -11,18 +11,226 @@
...
+
+uint DBusDockInterface::windowSizeEfficient()
+{
+    return qvariant_cast<uint>(property("WindowSizeEfficient"));
+}
+
+void DBusDockInterface::setWindowSizeEfficient(uint value)
+{
+
+   setProperty("WindowSizeEfficient", QVariant::fromValue(value));
```
Comment (BLumia) [inline src/dbusinterface/dbusdockinterface.cpp:193]:
```suggestion
    setProperty("WindowSizeEfficient", QVariant::fromValue(value));
```

File: src/dbusinterface/dbusdockinterface.cpp [hunk 5]
```diff
@@ -11,18 +11,226 @@
...
+
+uint DBusDockInterface::iconSize()
+{
+    return qvariant_cast<uint>(property("IconSize"));
+}
+
+void DBusDockInterface::setIconSize(uint value)
+{
+
+   setProperty("IconSize", QVariant::fromValue(value));
```
Comment (BLumia) [inline src/dbusinterface/dbusdockinterface.cpp:140]:
```suggestion
    setProperty("IconSize", QVariant::fromValue(value));
```

File: src/dbusinterface/dbusdockinterface.cpp [hunk 6]
```diff
@@ -11,18 +11,226 @@
...
 
+double DBusDockInterface::opacity()
+{
+    return qvariant_cast<double>(property("Opacity"));
+}
+
+void DBusDockInterface::setOpacity(double value)
+{
+
+   setProperty("Opacity", QVariant::fromValue(value));
```
Comment (BLumia) [inline src/dbusinterface/dbusdockinterface.cpp:151]:
```suggestion
    setProperty("Opacity", QVariant::fromValue(value));
```

**Review by BLumia — COMMENTED** [review]
需要测试在 topic 仓库里验证吧。顺便，dock 有没有考虑直接把这些接口提供成 C++ 库？

**Comment by tsic404** [comment]
daemon.Dock1接口保持不变，暂不需要更新

**Pull request closed without merging**